src/box/box.cc

```diff
@@ -797,14 +823,136 @@ wal_stream_apply_dml_row(struct wal_stream *stream, struct xrow_header *row)
 	return -1;
 }
 
+/* Storage, adding rows to a rlist to separate mixed transactions. */
+struct wal_row {
```

Comment:
I couldn't parse the comment. Could you please rephrase?

Reply:
Fixed.